journals/apps/journals/models.py

```diff
@@ -176,20 +176,37 @@ def __str__(self):
         return str(self.uuid)
 
     @classmethod
-    def user_has_access(cls, user, journal):
-        """ Checks if the user has access to this journal """
+    def get_user_accessible_journals(cls, user):
```

Comment:
proposing this method returns a Queryset.values_list() of journal ids

Reply:
Fixed